ldap/servers/slapd/back-ldbm/db-bdb/bdb_misc.c

```diff
@@ -383,8 +422,7 @@ bdb_start_autotune(struct ldbminfo *li)
         slapi_log_err(SLAPI_LOG_WARNING, "bdb_start_autotune", "Total entry cache size: %" PRIu64 " B; dbcache size: %" PRIu64 " B; available memory size: %" PRIu64 " B; \n",
                       total_cache_size, (uint64_t)li->li_dbcachesize, mi->system_available_bytes);
         slapi_log_err(SLAPI_LOG_WARNING, "bdb_start_autotune", "%s\n", msg);
-        /* WB 2016 - This should be UNCOMMENTED in a future release */
-        /* return SLAPI_FAIL_GENERAL; */
+        return SLAPI_FAIL_GENERAL;
```

Comment:
Not sure it should fail. It is just a warning but we may let the admin set this value.

Reply:
On second thought, my feeling is that we should keep the original behavior without returning.
If we decide to change the behavior we should better to do it in another PR
 (especially if we need to undo it afterwards ! :wink: )